conjur/http.py

```diff
@@ -60,7 +60,18 @@ def invoke_endpoint(http_verb, endpoint, params, *args, check_errors=True,
                               headers=headers)
 
     if check_errors:
-        response.raise_for_status()
+        # takes the "requests" response object and expands the
+        # raise_for_status method to return more helpful errors for debug logs
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as http_error:
+            if response.text:
+                logging.debug(requests.exceptions.HTTPError(f'{http_error.response.status_code}' \
```

Comment:
it should be logging.error or logging.exception

Reply:
I want this to be returned to the user in debug and not be printed to regular stdout